src/puppetlabs/puppetdb/config.clj

```diff
@@ -356,19 +361,22 @@
 
 (defn add-web-routing-config
   [config-data]
-  (let [prefix (normalize-url-prefix (get-in config-data [:global :url-prefix]
-                                             ""))]
-    (assoc-in config-data [:web-router-service
-                           :puppetlabs.puppetdb.cli.services/puppetdb-service] prefix)))
+  (let [url-prefix-path [:web-router-service :puppetlabs.puppetdb.cli.services/puppetdb-service]
+        prefix (or (get-in config-data url-prefix-path)
+                   ;; Setting url-prefix in [global] is deprecated
+                   (normalize-url-prefix (get-in config-data [:global :url-prefix]))
+                   "")]
```

Comment:
@rbrw I know we discussed the fact that the two config options needed different defaults but I think the the one we we're using for the web-routing config `"/"` was just wrong after reading the tk docs. `"/"` has the desired behavior (mounting the app with no prefix) for trapperkeeper but not for `compojure/context` which gives the app the prefix `/`. If you're mounting the app without a prefix you probably won't be using the option at all, while if you are mounting with a prefix you'll probably be using something more than "/" such as "/pdb".